web/src/components/retro-show/retro_cable.test.jsx

```diff
@@ -32,17 +32,18 @@
 import React from 'react';
 import {mount} from 'enzyme';
 import ActionCable from 'actioncable';
-import {Dispatcher} from 'p-flux';
 import '../../spec_helper';
 
 import RetroCable from './retro_cable';
 
 describe('RetroCable', () => {
   let retroCableDOM;
+  let websocketRetroDataReceived;
 
   beforeEach(() => {
+    websocketRetroDataReceived = jest.fn();
     const cable = ActionCable.createConsumer('wss://websocket/url');
-    retroCableDOM = mount(<RetroCable cable={cable} retro_id="retro-slug-123"/>);
+    retroCableDOM = mount(<RetroCable cable={cable} retro_id="retro-slug-123" websocketRetroDataReceived={websocketRetroDataReceived}/>);
```

Comment:
Is there any way to test whether `websocketRetroDataReceived` gets invoked at the right time?

Reply:
Yea, think I removed the 'test' for this as I didn't think it really tested that. I would probably revisit this if I get round to writing some component integration tests.